deciders/correctness-deciders.tex

```diff
@@ -24,13 +24,14 @@
 
 \usepackage{amsmath,amsfonts,amssymb,amsthm}
 
-
+\theoremstyle{definition} % don't use italics
 \newtheorem{theorem}{Theorem}
 \newtheorem{definition}[theorem]{Definition}
 \newtheorem{lemma}[theorem]{Lemma}
 \newtheorem{proposition}[theorem]{Proposition}
 \newtheorem{corollary}[theorem]{Corollary}
 
+\theoremstyle{remark} % emphasize the "Remark N." with italics, not bold
```

Comment:
Inspired by the existence of `theoremstyle`, I changed the formatting of the remark-like environments. That is arguably a more subjective matter. Apart from that, the PDF should render the same.

Reply:
Yeah, I also dislike that style. It makes the inline equations much less visually distinct, and `\emph` looks weird when you need it.

Reply:
Ok ahah, I think we can stick to this style (anyway we might have to comply to some other style for conference/journal submissions).

However, a point that was brought up by savask and that we should probably change in the current paper is the number system for def/theorems/remarks/examples/etc and have a counter per section VS a unique counter in the document. But that can be handled in another PR.